### ISSUE TITLE
Update HydroShare Landing Page 

### ISSUE THREAD

**Issue by horsburgh** [hydroshare/hydroshare#4297]
**Describe the feature you'd like and what it will do**

We want to update the main HydroShare landing page to include new information about how HydroShare can support research and who is using HydroShare.

**Why is this feature important?**
A short description of the importance of this feature and what it will help you achieve.

This will answer potential user questions about how they can use HydroShare and assures users that HydroShare is being used by reputable organizations and is trusted by the scientific community.

**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...].

This is a reorganization of existing information on the HydroShare landing page along with addition of new, user-focused information. No new functionality is introduced.

**Additional context**

The mockup of the desired content and layout is here: https://app.moqups.com/foPTDLxgLR/view/page/ad64222d5

**Comment by pkdash** [comment]
@horsburgh and @dtarb I think @Maurier can implement this UI work much quicker than me.

**Comment by Maurier** [comment]
@pkdash Right, I'll take this one.

**Comment by horsburgh** [comment]
@Maurier - here is a list of URLs for the icons at the bottom of the page:

Under Who Uses HydroShare?
* iUTAH: https://www.hydroshare.org/group/6
* CZNet: https://www.criticalzone.org/ (we'll eventually change this when the CZ Net community goes live in HydroShare
* CZO: https://www.hydroshare.org/community/1/
* Logan River Observatory: https://www.hydroshare.org/group/138

Under Trusted By the Scientific Community
* https://fairsharing.org/
* https://www.re3data.org/
* https://www.springernature.com/gp
* JGR Biogeosciences: https://agupubs.onlinelibrary.wiley.com/journal/21698961
* Scientific Data: https://www.nature.com/sdata/policies/repositories
* I couldn't find a specific icon for ASCE Journal of Water Resources Planning and Management, but the website is here: https://ascelibrary.org/page/dataavailability. You could probably use the ASCE Library logo because it's the same for all ASCE journals.
* AGU: https://data.agu.org/resources/useful-domain-repositories use an AGU logo like the one that appears at the top of this page
* COPDESS: https://copdess.org/ I guess use the wordmark at the top of the page

**Comment by Maurier** [comment]
The new content can be seen here: https://new-testservices.hydroshare.org/

**Comment by dtarb** [comment]
@horsburgh I'd like to make sure that all the links we have under "Under trusted" are justifiably legitimate.  

1. What is the basis for listing Springer Nature.  If I search at that URL for hydrology, or hydroshare, I do not find anything about hydroshare.
2. What legitimizes JGR Biogeosciences over any other AGU Journal?  As JGR is published by AGU, I think we should only mention AGU and not JGR.

**Comment by horsburgh** [comment]
@dtarb - I dug into the recommended repositories by both of those journals, and both mention HydroShare. 

For Springer Nature, go here: https://www.springernature.com/gp/authors/research-data-policy/recommended-repositories then click on "Earth, Environmental and Space Sciences", then click on "Broad Scope Earth and environmental sciences". HydroShare is listed there.

For JGR BioGeosciences, go to this page: https://data.agu.org/resources/useful-domain-repositories scroll down or search for HydroShare to find its listing on the page and it specifically names WRR, JGR BioGeosciences, and JGR Oceans.

**Comment by dtarb** [comment]
@horsburgh Fair enough.  But, I just think that what is currently featured is not what resonates with, at least my perception of the HydroShare resources.  WRR is the AGU hydrology section journal.  If we are to list any AGU journal it should be WRR.   Perhaps we should put the AGU logo since we are listed under data.agu.org.  

Under research organizations, much as I like iUtah, this is a project that is over.  This is our front page, I think we need to plan carefully, and it seems like what is there is somewhat random at the moment.

**Comment by horsburgh** [comment]
@dtarb - I took the content I was given by Jon Pollak and put it on the page. I honestly can't remember now whether I added additional things to what he originally gave me or not. If you want us to change something, just tell us what you want.

With regard to featuring iUTAH - sure that project is over, and we can remove it if you want. But, so are the CZO projects. The corpus of CZO data that now resides in HydroShare is a pretty valuable resource that I think we should promote. HydroShare is an archival repository, so perhaps it's not strictly accurate to feature these under "Who uses HydroShare" - but they are assets that I think we can and should promote.
